lib/main/STM32F7/Middlewares/ST/STM32_USB_Device_Library/Class/HID/Inc/usbd_hid.h

```diff
@@ -46,7 +46,7 @@
 
 #define USB_HID_CONFIG_DESC_SIZ       34U
 #define USB_HID_DESC_SIZ              9U
-#define HID_MOUSE_REPORT_DESC_SIZE    38U
+#define HID_MOUSE_REPORT_DESC_SIZE    54U
```

Comment:
Any background on this?
Such as: https://eleccelerator.com/tutorial-about-usb-hid-report-descriptors/

Reply:
This is the bytes count which will be sent to the driver. It expects right number of bytes and if it's missed, USB will yell massive error report.
![image](https://user-images.githubusercontent.com/7397515/91671166-6be8f280-eb24-11ea-8213-e1bd2504c248.png)

Reply:
Makes all sense now :-D